### ISSUE TITLE
illust_bookmark_add(), illust_bookmark_delete(), user_list() error in aapi.py

### ISSUE THREAD

**Issue by naplings** [upbit/pixivpy#39]
## Issue:  
[**illust_bookmark_add()**](https://github.com/upbit/pixivpy/blob/master/pixivpy3/aapi.py#L254), [**illust_bookmark_delete()**](https://github.com/upbit/pixivpy/blob/master/pixivpy3/aapi.py#L270), [**user_list()** ](https://github.com/upbit/pixivpy/blob/master/pixivpy3/aapi.py#L328)doesn't work correctly.
Following script:  
```python
api = AppPixivAPI()
api.login('LOGIN INFORMATION')
print(api.illust_bookmark_add(62345606))
print(api.illust_bookmark_delete(62345606))
print(api.user_list(3410615))
```
Output:
```
{'error': {'user_message': '指定されたエンドポイントは存在しません', 'message': '', 'reason': '', 'user_message_details': {}}}
{'error': {'user_message': '', 'message': '{"illust_id":["Illust Id is required","Illust Id must be an integer (0-9)","Illust
Id must be at least 0"]}', 'reason': '', 'user_message_details': {}}}
{'error': {'user_message': '指定されたエンドポイントは存在しません', 'message': '', 'reason': '', 'user_message_details': {}}}
```
I tried to change the URL in **user_list()** to V2 version,  
from:
```python
url = 'https://app-api.pixiv.net/v1/user/list'
```
to:
```python
url = 'https://app-api.pixiv.net/v2/user/list'
```
And it works.  
However, it doesn't work on **illust_bookmark_add()** and **illust_bookmark_delete()**.

## Environment:
`Python 3.6.0`
`pixivpy3 '3.3.1'`
`Microsoft Windows [Ver 10.0.16176]`

## 
At last, thanks for your awesome library. It helps me pretty much.

**Comment by upbit** [comment]
`I tried to change the URL in user_list() to V2 version,`
En, may be Pixiv move those endpoint from v1/ to v2/

Thanks for feedback! I will sniffer the new API as soon as possible.

**Comment by naplings** [comment]
Thank you so much.  
 There is another thing confused me that [illust_bookmark_detail()](https://github.com/upbit/pixivpy/blob/master/pixivpy3/aapi.py#L245) is requiring for OAuth process. Changing `req_auth = False` to `req_auth = True` can get the correct response rather than the "error json". I'm not 100% sure whether it would happen on my Windows or not. I only tested it on mac.

**Comment by upbit** [comment]
`illust_bookmark_detail()` now require OAuth for all illusts, previously only R18 illusts needed.

**Comment by upbit** [comment]
Sorry for may fault, fixed in [v3.3.3](https://github.com/upbit/pixivpy/commit/41eefc4336fc7bcf666ea2d7695dcf619aaccc78), please upgrade `pip install pixivpy --upgrade` and try this:

```py
    api = AppPixivAPI()
    try:
        print(api.illust_bookmark_detail(62345606))
    except:
        print("illust_bookmark_detail() will raise when not login")

    api.login(_USERNAME, _PASSWORD)   # Not required

    print(api.illust_bookmark_add(62345606))
    print(api.user_list(3410615))
    print(api.illust_bookmark_delete(62345606))
```

```
illust_bookmark_detail() will raise when not login
{}
{'users': []}
{}
```
